### PR TITLE
Add restore_fallback model arg

### PR DESCRIPTION
**Pull request by blazekadam**
**cxflow** saves `model.restore_fallback` to the config file which prevents `cxflow train ...` from working as the `restore_fallback` argument is not handled by the `BaseModel`. 

(`cxflow resume ...` worked fine).

This PR fixes this behavior.

### CODE REVIEW

**Comment by blazekadam** [comment]
@petrbel @FloopCZ can you review this?

File: cxflow_tensorflow/model.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def __init__(self,  # pylint: disable=too-many-arguments
                  dataset: Optional[cx.AbstractDataset], log_dir: Optional[str], inputs: List[str], outputs: List[str],
                  session_config: Optional[dict]=None, n_gpus: int=0, restore_from: Optional[str]=None,
                  optimizer=None, freeze=False, loss_name: str=DEFAULT_LOSS_NAME, monitor: Optional[str]=None,
+                 restore_fallback: Optional[str]=None,
```
Comment (FloopCZ) [inline cxflow_tensorflow/model.py]: It should be documented on L82. Especially since this is a dummy parameter that is actually never used.

Comment (blazekadam) [reply cxflow_tensorflow/model.py]: Excellent point. Done.

File: cxflow_tensorflow/model.py [hunk 2]
```diff
@@ -147,6 +149,7 @@ def __init__(self,  # pylint: disable=too-many-arguments
             logging.debug('\tSearching for the train ops in the created graph')
             try:
                 for i in range(len(self._towers)):
+
```
Comment (FloopCZ) [inline cxflow_tensorflow/model.py]: (Intentional newline?)

Comment (blazekadam) [reply cxflow_tensorflow/model.py]: What newline :)

**Pull request merged**